src/app.js

```diff
@@ -7,8 +7,12 @@ import { loadMovieReviews } from './movieReviews.js';
 import { getLatestScreenings } from './latestScreeningsFromAPI.js';
 import { getSpecificScreenings } from './specificScreeningsFromApi.js';
 import cmsAdapter from './cmsAdapter.js';
+import { review } from './reviewBody.js';
 
 const app = express();
+app.use(express.urlencoded({ extended: true }));
```

Comment:
I don't think this is used anymore. If I'm correct, it can be deleted. It's always good to get rid of "dead code" right away, because the more time passes the more difficult it's going to be to remember if it's really dead or why it was ever there.